src/arviz_plots/plots/psensedistplot.py

```diff
@@ -0,0 +1,292 @@
+    else:
+        figsize, figsize_units = plot_bknd.get_figsize(plot_collection)
+        aux_dim_list = list(
+            set(
+                dim for child in plot_collection.viz.children.values() for dim in child["plot"].dims
+            ).difference({"column"})
+        )
+        row_dims = ["__variable__"] + aux_dim_list
+
+    figsize = plot_bknd.scale_fig_size(
+        figsize,
+        rows=process_facet_dims(distribution, row_dims)[0],
+        cols=2,
+        figsize_units=figsize_units,
+    )
+
+    color_cycle = pc_kwargs.get("color", plot_bknd.get_default_aes("color", 3, {}))
+    if len(color_cycle) <= 2:
+        raise ValueError(
+            f"Not enough values provided for color cycle, got {color_cycle} "
+            "but at least 3 are needed"
+        )
+
+    plot_kwargs.setdefault("point_estimate_text", False)
+
+    # Middle chain is the reference chain (alpha == 1)
+    pc_kwargs.setdefault("color", [color_cycle[0], "k", color_cycle[1]])
+    pc_kwargs.setdefault("y", [-0.4, -0.225, -0.05])  # XXX can we use relative values?
+    pc_kwargs.setdefault("aes", {"color": ["chain"], "y": ["chain"]})
+
+    if plot_collection is None:
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+        pc_kwargs.setdefault("cols", ["column"])
+        pc_kwargs["plot_grid_kws"] = pc_kwargs.get("plot_grid_kws", {}).copy()
+        if "figsize" not in pc_kwargs["plot_grid_kws"]:
+            pc_kwargs["plot_grid_kws"]["figsize"] = figsize
+            pc_kwargs["plot_grid_kws"]["figsize_units"] = "dots"
+
+        pc_kwargs["plot_grid_kws"].setdefault("sharex", "row")
+        pc_kwargs["plot_grid_kws"].setdefault("sharey", "row")
+
+        plot_collection = PlotCollection.grid(
+            distribution.expand_dims(column=2).assign_coords(column=["prior", "likelihood"]),
+            backend=backend,
+            **pc_kwargs,
+        )
+
+    if aes_map is None:
+        aes_map = {}
+    else:
+        aes_map = aes_map.copy()
+
+    aes_map.setdefault("point_estimate", ["color", "y"])
+    aes_map.setdefault("credible_interval", ["color", "y"])
+
+    if labeller is None:
+        labeller = BaseLabeller()
+
+    if kind == "hist":
+        # Histograms are not great for overlapping distributions
+        # But "step" histograms may be slightly easier to interpret than bars histograms
+        # Using the same number of "bins" should help too
+        plot_kwargs.setdefault("hist", {})
+        plot_kwargs["hist"].setdefault("alpha", 0.3)
+        plot_kwargs["hist"].setdefault("edgecolor", None)
+        stats_kwargs.setdefault("density", {"density": True})
+
+    if alphas is None:
+        alphas = (0.8, 1.25)
+
+    # Here we are generating new datasets for the prior and likelihood
+    # by resampling the original dataset with the power scale weights
+    # Instead we could have weighted KDEs/ecdfs/etc
+    ds_prior = new_ds(dt, "log_prior", alphas)
+    ds_likelihood = new_ds(dt, "log_likelihood", alphas)
```

Comment:
I would actually merge these two datasets along a `component` dimension which is what I'd use as the facetting dimension for the columns. Then a single call to `plot_dist` should be enough plus we'll automatically get the correct labels in the titles, with also the changes in `new_ds` we'd have to hardcode `sample_dims="sample"` when calling `plot_dist` but that is all.

I'd also do that before initializing the plotcollection so we can use `cols=["component"]` and `aes={"color": ["alpha"]}` when defining mappings.